bindings/nodejs/types/output.ts

```diff
@@ -4,6 +4,11 @@ import type {
     IOutputMetadataResponse,
 } from '@iota/types';
 
+/**
+ * The identifier of an Output
+ */
+export type OutputId = string;
```

Comment:
Is it worth defining a OutputId as a new string type? I feel like it's a bit of an overkill, since we could just use string everywhere.

Reply:
It's just a personal preference, with a better comment it gives an opportunity to convey a bit more information than that it is a string. It is a string, but it's different from for example `blockId` which is also a string. Implementing this for OutputId means it should also be implemented for the other ID types though. I did this for `BlockId` in [iota.rs](https://github.com/iotaledger/iota.rs/blob/develop/bindings/nodejs/types/blockId.ts), but not the other types, so I should go back there and reconsider adding the other id types or removing `BlockId`. 

It's just an alias and doesn't do anything to verify that what is passed in is a valid OutputId. That can be done in typescript, but it is cumbersome and not without caveats, so I don't think that should be implemented. 

I don't feel strongly about this either way though, so let me know and I'll happily revert it. Thanks!

Reply:
Thanks for your explanation (and your help with this!),  it makes sense. Let's keep it like that then :)!